components/linkButton.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {Linking, StyleSheet, Text, View} from 'react-native';
+import {Colors, GlobalStyles} from '../styles';
+
+export default function LinkButton({title, url}) {
```

Comment:
If I'm not mistaken, you don't need to pass arguments as an object and can simply do: `LinkButton(title, url)` if you prefer

Reply:
The syntax as written by @drumwolf is correct- this is a destructuring assignment of the props object.